charts/appswitcher-server/Chart.yaml

```
@@ -5,9 +5,9 @@ type: application
 version: 1.2.5
 appVersion: "1.3.0"
 maintainers:
-  - name: eidottermihi
-    email: eidottermihi@gmail.com
+- name: eidottermihi
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 1

(indentation)